### PR TITLE
Feature/us7911 get out of my stuff view

### PR DESCRIPTION
**Pull request by vchtelmakh**

### CODE REVIEW

File: src/app/components/map-footer/map-footer.component.spec.ts [hunk 1]
```diff
@@ -76,4 +88,10 @@ describe('Component: MapFooter', () => {
         expect(comp).toBeTruthy();
     });
 
+    it('should change state to my stuff if it is inactive and "My Stuff" is clicked', () => {
+        spyOn(comp, 'changeStateToMyStuff');
```
Comment (JoeKer1) [inline src/app/components/map-footer/map-footer.component.spec.ts]: Not sure this really tests anything other than that you called the function

Comment (vchtelmakh) [inline src/app/components/map-footer/map-footer.component.spec.ts]: It tests that the correct function was called - a different function is called if my stuff is active.

File: src/app/components/map-footer/map-footer.component.spec.ts [hunk 2]
```diff
@@ -16,6 +16,16 @@ import { LoginRedirectService } from '../../services/login-redirect.service';
 import { UserLocationService } from  '../../services/user-location.service';
 import { AppSettingsService } from '../../services/app-settings.service';
 
+function fakenext(param: any) { return 1; }
```
Comment (JoeKer1) [inline src/app/components/map-footer/map-footer.component.spec.ts]: I might have a way this actually works, I haven't tested it yet though. It works in jest 🥇

File: src/app/components/search-bar/search-bar.component.html [hunk 3]
```diff
@@ -2,10 +2,14 @@
...
       <input type="text"
-              class="form-control"
-              [(ngModel)]="searchText"
-              (keyup.enter)="onSearch(searchText)"
-              placeholder="Search zip or keyword...">
+             class="form-control"
+             [(ngModel)]="searchText"
+             (keyup.enter)="onSearch(searchText)"
+             placeholder="Search zip or keyword...">
+      <!--(keyup) = "searchKeyUp()"--> <!--This attribute belongs to the input above-->
```
Comment (JoeKer1) [inline src/app/components/search-bar/search-bar.component.html]: can this be removed?

File: src/app/services/state.service.ts [hunk 4]
```diff
@@ -32,7 +32,7 @@ export class StateService {
   private myViewOrWorldView: string = 'world';
   private zoomToUse: number = -1;
   private savedMapView: MapView;
-  private lastSearch: SearchOptions;
+  public lastSearch: SearchOptions;
```
Comment (JoeKer1) [inline src/app/services/state.service.ts]: Should this have a getter like the other things? I dunno probably doesn't matter.

File: src/app/components/search-bar/search-bar.component.html (hunk 3 above)
Comment (vchtelmakh) [inline src/app/components/search-bar/search-bar.component.html]: I would rather not, Ample is going to use it soon.

File: src/app/components/map-footer/map-footer.component.spec.ts (hunk 2 above)
Comment (vchtelmakh) [inline src/app/components/map-footer/map-footer.component.spec.ts]: Let me know.

**Pull request merged**